R/get_coding_ssm.R

```diff
@@ -34,24 +36,25 @@
 #' library(dplyr)
 #' 
 #' #return SSMs in reference to GRCh37:
-#' ssm_grch37 = get_coding_ssm(seq_type = "genome")
+#' ssm_grch37 = get_coding_ssm(this_seq_type = "genome")
 #'
 #' #return SSMs in reference to hg38:
 #' cell_line_meta = GAMBLR.data::sample_data$meta %>% 
 #'   dplyr::filter(cohort == "DLBCL_cell_lines")
 #' 
 #' ssm_hg38 = get_coding_ssm(projection = "hg38", 
 #'                           these_samples_metadata = cell_line_meta, 
-#'                           seq_type = "genome")
+#'                           this_seq_type = "genome")
 #' 
 get_coding_ssm = function(limit_cohort,
                           exclude_cohort,
                           limit_pathology,
                           limit_samples,
+                          these_sample_ids,
                           these_samples_metadata,
                           force_unmatched_samples,
                           projection = "grch37",
-                          seq_type = "genome",
+                          this_seq_type = "genome",
```

Comment:
I think in the master GAMBLR it is called `seq_type` so this renaming might not work as expected. Same for the version of this function in GAMBLR.results, this parameter is without `this_` prefix

Reply:
Yes, this is true. For this repo, I decided to harmonize the parameter names (this_seq_type) to improve the intuitiveness of how to use this parameter. But we can obviously revert and have the parameter names consistent throughout both repos. But I would rather vote for harmonizing the same parameters in GAMBLR.results the same way as I did for this repo.

Reply:
I agree, harmonizing the naming is the best way to go and these can remain with the changes as long as the parallel version in the GAMBLR.results is also updated 👍

Reply:
Good, I can make an issue of this in GAMBLR.results for tracking purposes. Thanks!